pkg/yurtctl/cmd/clusterinfo/clusterinfo.go

```diff
@@ -30,7 +30,7 @@ import (
 	"k8s.io/klog/v2"
 
 	"github.com/openyurtio/openyurt/pkg/projectinfo"
-	kubeutil "github.com/openyurtio/openyurt/pkg/yurtctl/util/kubernetes"
+	kubeutil "github.com/openyurtio/openyurt/pkg/yurtadm/util/kubernetes"
```

Comment:
It looks strange that yurtctl import package of yurtadm.
Is it possible to move `pkg/yurtadm/util/kubernetes` to `pkg/util/kubernetes`

Reply:
It can be, but `pkg/util/kubernetes` also depend on `pkg/yurtadm/kubernetes`.

Reply:
What do you mean? If we move `pkg/yurtadm/util/kubernetes` entirely to `pkg/util/kubernetes`, there's no dependence between them.

Reply:
There's a lot of subtle dependencies between them. For example, `pkg/yurtadm/kubernetes/kubeadm/app/discovery/token` depend on `pkg/yurtadm/cmd/join/joindata`. And `pkg/yurtadm/kubernetes` and `pkg/yurtadm/util/kubernetes` are different files. :(

Reply:
Well, that's a problem. I find that yurtctl depends on the `yurtadm/kubernetes` mostly beacuse of convert/revert and test cmd. And test relies on `yurtadm/kubernetes` beacuse it reuses the code of convert. It's a solution that move test into yurtadm and remove convert/revert subcommand(which is under work in pr #748).

For dependence between `pkg/yurtadm/kubernetes/kubeadm/app/discovery/token` and `pkg/yurtadm/cmd/join/joindata`, can we move the joindata into `yurtadm/kubernetes`? It seems that `yurtadm/kubernetes` only depends on joindata pkg.

And what do you think @rambohe-ch ?

Reply:
@Congrool @lonelyCZ The original idea of `yurtadm/kubernetes` or `yurtctl/kubernetes` is that codes comes from kubernetes project in order to avoid to import `k8s.io/kubernetes`. so move `joindata` into `yurtadm/kubernetes` maybe change the original idea.

how about deprecate `yurtctl convert/revert` at first?  and now we have no response form writer of #748, so maybe we can post a new pull request to solve this case.  @lonelyCZ would you like to take over this case?

Reply:
Ok.